File: translations/codenet/mistral_commenetd/C2J/codellama/Code/s423495369.java

// Declare three integer variables A, B, and C
public class Main {
  public static void main(String[] args) {
    int A, B, C;

    // Read three integers from the standard input using Scanner class
    Scanner scanner = new Scanner(System.in);
    A = scanner.nextInt();
    B = scanner.nextInt();
    C = scanner.nextInt();

    // Check if A is less than C and C is less than B
    if (A < C && C < B) {
      // If condition is true, print "Yes" to the standard output
      System.out.println("Yes");
    } // End of first if block

    // Else if condition checks if A is greater than C and C is greater than B
    else if (A > C && C > B) {
      // If condition is true, print "Yes" to the standard output
      System.out.println("Yes");
    } // End of second if block

    // Else condition is executed when neither first nor second condition is true
    else {
      // If condition is false, print "No" to the standard output
      System.out.println("No");
    } // End of else block
  } // End of main function
} // End of Main class

File: translations/codenet/mistral_commenetd/C2P/gpt-4o/Code/s423495369.py
# Declare three integer variables A, B, and C
def main():
    # Read three integers from the standard input
    A, B, C = map(int, input().split())

    # Check if A is less than C and C is less than B
    if A < C < B:
        # If condition is true, print "Yes" to the standard output
        print("Yes")
    # Else if condition checks if A is greater than C and C is greater than B
    elif A > C > B:
        # If condition is true, print "Yes" to the standard output
        print("Yes")
    # Else condition is executed when neither first nor second condition is true
    else:
        # If condition is false, print "No" to the standard output
        print("No")

# Call the main function
if __name__ == "__main__":
    main()

# <END-OF-CODE>
